discovery-infra/tests/test_wrong_boot_order.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from tests.base_test import BaseTest
+
+
+class TestWrongBootOrder(BaseTest):
+    @pytest.mark.regression
+    def test_wrong_boot_order_one_node(self, api_client, node_controller, cluster):
+        # Define new cluster
+        cluster_id = cluster().id
+        # Generate and download cluster ISO
+        self.generate_and_download_image(cluster_id=cluster_id, api_client=api_client)
+        # Change boot order of a node
+        hosts = node_controller.list_nodes()
+        node = list(hosts.keys())[0]
+        node_controller.set_boot_order_to_node(node_name=node, cd_first=True)
```

Comment:
as i said, i want to minimize usage of node_controller.
we need to add class Hosts that will have list of hosts and it will start relevant node_controller.
Didn't have time to add it